CARDS-1852: Bold fonts in Firefox are not bold enough

sdumitriu opened the pull request:
Draft PR; to test: build, run, does the page look too heavy?

sdumitriu committed:
CARDS-1852: Bold fonts in Firefox are not bold enough

marta- commented:
Nope, nope. I like the 300 default especially in the Questionnaire wizard, please don't tinker with it.

sashaandjic commented:
Why we have scrollbars in Chrome and not in FF?
![image](https://user-images.githubusercontent.com/23243244/177672989-9dd0bf07-1a0a-4ac4-8335-de154e32b63d.png)

marta- commented:
> Why we have scrollbars in Chrome and not in FF? 

Good question, but it's certainly not because of this PR.

This only changes the default font weight, but it changes it **globally**. In addition to the MDEditor, the fonts in livetables, materialtables, and questionnaire wizard values will be affected (probably others as well).

marta- committed:
CARDS-1852: Bold fonts in Firefox are not bold enough
Restore thin font weight for questionnaire item property values in the questionnaire editor

marta- reviewed (approved):
The "not bold enough" in FF is solved.

sashaandjic reviewed (approved):
tested
![image](https://user-images.githubusercontent.com/23243244/177812895-8faa820a-88a0-498d-a598-c0b98f7dd9d3.png)
